packages/workflow/src/documents.ts

```diff
@@ -18,36 +18,23 @@ import {
 } from '@opencrvs/commons/types'
 import { CertifyInput, IssueInput } from './records/validations'
 
-const fetchDocuments = async <T = any>(
-  suffix: string,
+export async function uploadFileToMinio(
```

Comment:
Better to split this into two functions `uploadFileToMinio` and `uploadSVGToMinio`

Reply:
Agreed with Riku. Otherwise the changes look good :+1: